auto_submit/lib/service/config.dart

```diff
@@ -48,6 +48,9 @@ class Config {
   /// Labels autosubmit looks for on pull requests
   static const String kAutosubmitLabel = 'autosubmit';
 
+  /// GitHub check stale threshold.
+  static const int kGitHubCheckStaleThreshold = 2; // hours
```

Comment:
I wonder if this would be something to put into the autosubmit configuration as then a repository could specify if they want to change the stale check value. For example in cocoon it could 1 hour or in flutter it could be 3. Something to think about long term as things may change. Also keep in mind we support dart-lang as well.

Reply:
Yeah, this is a good point. 
But the main motivation of this PR is to help ease engine gardener manually monitoring these roller PRs.
Non-engine roller PRs are expected to be monitored by their authors. I would prefer to focusing on these roller PRs.
If people find it useful to be alerted as well, then we can easily extend the logic by posting a comment.

To begin with, I updated the logic to be limited to engine roller specific.

Reply:
Yeah no problem. I am just thinking out a couple of quarters. I have some changes in mind to make to cleanup autosubmit further so not a big deal here.